lib/src/multires_stochastic_texture_synthesis.rs

```diff
@@ -963,7 +968,6 @@ impl Generator {
             {
                 let actual_total_pixels_to_resolve: usize =
                     (0..=params.p_stages).map(stage_pixels_to_resolve).sum();
-                let mut total_processed_pixels = 0;
```

Comment:
This got put in the wrong scope, to make it cumulative again we need to bump it up a level.